CMakeLists.txt

```diff
@@ -55,45 +59,49 @@ endif()
 ####################################################
 # Build Options
 ####################################################
-SET(BUILD_TESTS YES CACHE BOOL "Build test applications")
+
+option(BUILD_TESTS "Build test applications" YES)
 
 if(EXISTS "${CMAKE_CURRENT_SOURCE_DIR}/wolfssl/options.h")
     # Copy generated ./options.h
     configure_file(${CMAKE_CURRENT_SOURCE_DIR}/wolfssl/options.h 
-                   ${CMAKE_CURRENT_BINARY_DIR}/user_settings.h)
+                   ${CMAKE_CURRENT_SOURCE_DIR}/user_settings.h)
 else()
    # Use template
    configure_file(${CMAKE_CURRENT_SOURCE_DIR}/wolfssl/options.h.in 
-                  ${CMAKE_CURRENT_BINARY_DIR}/user_settings.h)
+                  ${CMAKE_CURRENT_SOURCE_DIR}/user_settings.h)
 endif()
 
 add_definitions(-DWOLFSSL_USER_SETTINGS)
 add_definitions(-DWOLFSSL_IGNORE_FILE_WARN)
-if(CMAKE_HAVE_PTHREAD_H)
+if(CMAKE_USE_PTHREADS_INIT)
```

Comment:
I believe this is the more up-to-date variable, based on this: https://cmake.org/cmake/help/v3.19/module/FindThreads.html

Reply:
Thanks